container/crio/handler.go

```diff
@@ -305,6 +305,14 @@ func (h *crioContainerHandler) GetStats() (*info.ContainerStats, error) {
 	if err != nil {
 		return stats, err
 	}
+
+	if len(stats.Network.Interfaces) == 0 {
+		// No network related information indicates that the pid of the
+		// container is not longer valid and we need to ask crio to
+		// provide the pid of another container from that pod
+		h.pidKnown = false
+		return stats, nil
+	}
```

Comment:
I think we should double check `h.needNet()` here, or else we may needlessly recall into cri-o because a container didn't provide the network stats

Reply:
we can just combine this check as  `else if len(...)` below

Reply:
@haircommander Thanks, good point. Updated.